[2023-02-07 18:22:13,582] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 07.1_external_task.external_task_1 scheduled__2022-11-15T00:00:00+00:00 [queued]>
[2023-02-07 18:22:13,596] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 07.1_external_task.external_task_1 scheduled__2022-11-15T00:00:00+00:00 [queued]>
[2023-02-07 18:22:13,597] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-07 18:22:13,597] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-07 18:22:13,598] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-07 18:22:13,624] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): external_task_1> on 2022-11-15 00:00:00+00:00
[2023-02-07 18:22:13,627] {standard_task_runner.py:52} INFO - Started process 31202 to run task
[2023-02-07 18:22:13,633] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '07.1_external_task', 'external_task_1', 'scheduled__2022-11-15T00:00:00+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/07.1_external_task.py', '--cfg-path', '/tmp/tmp5pctgrz2', '--error-file', '/tmp/tmplovdpgbp']
[2023-02-07 18:22:13,634] {standard_task_runner.py:80} INFO - Job 432: Subtask external_task_1
[2023-02-07 18:22:13,725] {task_command.py:371} INFO - Running <TaskInstance: 07.1_external_task.external_task_1 scheduled__2022-11-15T00:00:00+00:00 [running]> on host pc.
[2023-02-07 18:22:13,823] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=07.1_external_task
AIRFLOW_CTX_TASK_ID=external_task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-11-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-15T00:00:00+00:00
[2023-02-07 18:22:13,825] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-07 18:22:13,826] {subprocess.py:74} INFO - Running command: ['bash', '-c', "sleep 10 && echo 'external task executed!'"]
[2023-02-07 18:22:13,833] {subprocess.py:85} INFO - Output:
[2023-02-07 18:22:23,836] {subprocess.py:92} INFO - external task executed!
[2023-02-07 18:22:23,836] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-07 18:22:23,877] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=07.1_external_task, task_id=external_task_1, execution_date=20221115T000000, start_date=20230207T232213, end_date=20230207T232223
[2023-02-07 18:22:23,925] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-07 18:22:23,947] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
